doc/rst/platforms/windows.rst

```diff
@@ -22,9 +22,41 @@ variety of Linux distributions. For more information on WSL, see the
 
 To use Chapel on WSL, you will need to install a Linux distribution from the
 Microsoft Store. We recommend using Ubuntu, but others will likely work.
-Once you have installed your distribution, get the list of prerequisites
-from :ref:`readme-prereqs` and install Chapel as you would on a native Linux
-system. There are no platform-specific settings for Chapel on WSL at this time.
+
+This example shows how to install WSL and Ubuntu on Windows 10/11::
+
+    # From PowerShell, run the following command to enable WSL
+    wsl --install -d Ubuntu
+
+
+There are two main approaches for using Chapel on WSL:
+
+1) Install via a prebuilt Chapel package. This is the quickest way to get up
+   and running, but it results in a copy of Chapel without GPU support and that
+   only supports shared-memory (single-locale) executions. See the list of available
+   packages released on the `Chapel GitHub page <https://github.com/chapel-lang/chapel/releases>`_.
+
+2) Build Chapel from source, as with any other UN*X system. This is slightly
```

Comment:
`UN*X` seems weird, what do we normally do to say unix?

Reply:
https://chapel-lang.org/docs/usingchapel/prereqs.html#readme-prereqs just says "UNIX".

Reply:
I think it came from the macos page: https://chapel-lang.org/docs/main/platforms/macosx.html. I can update them both if this is the oddball.

Reply:
No strong opinion here, but my assumption is that the prereqs page has more review, and so we should follow that.